ark/segmentation/marker_quantification.py

```diff
@@ -244,40 +244,40 @@ def generate_cell_data(segmentation_labels, tiff_dir, img_sub_folder,
         - arcsinh transformed data
     """
 
-    # if no points are specified, then load all the points
-    if points is None:
+    # if no fovs are specified, then load all the fovs
+    if fovs is None:
         # handle mibitiffs with an assumed file structure
         if is_mibitiff:
             filenames = io_utils.list_files(tiff_dir, substrs=['.tif'])
-            points = io_utils.extract_delimited_names(filenames, delimiter=None)
+            fovs = io_utils.extract_delimited_names(filenames, delimiter=None)
         # otherwise assume the tree-like directory as defined for tree loading
         else:
             filenames = io_utils.list_folders(tiff_dir)
-            points = filenames
+            fovs = filenames
 
-    # check segmentation_labels for given points (img loaders will fail otherwise)
-    point_values = [point for point in points if point not in segmentation_labels['fovs'].values]
-    if point_values:
-        raise ValueError(f"Invalid point values specified: "
-                         f"points {','.join(point_values)} not found in segmentation_labels fovs")
+    # check segmentation_labels for given fovs (img loaders will fail otherwise)
+    fov_values = [fov for fov in fovs if fov not in segmentation_labels['fovs'].values]
+    if fov_values:
```

Comment:
Like this for example, regarding my earlier comment. We can pull this out into its own function

Reply:
As you mentioned, probably best to create a PR called something like "Condense shared verification logic into test_utils.py" or something like that to address this. I'll add an issue for this.